src/exploreLondon/CommunityPartnerCard.js

```diff
@@ -41,8 +41,8 @@ const CommunityPartnerCard = ({ partner }) => {
     accessibilityDetails,
   } = partner
   return (
-    <Card>
-      <CardImg fixed={featuredImage.fixed} />
+    <StyledCard>
+      <StyledImage fixed={featuredImage.fixed} />
```

Comment:
The gap between image and categories is larger than previously - please compare to the live version of the page.

Reply:
Reverted 👍 .

Reply:
I left the same category filter as the one on https://prideinlondon.org/fifty-two but I can revert if needed.